src/engine/qcommon/net_chan.cpp

```diff
@@ -117,7 +117,7 @@ Send one fragment of the current message
 void Netchan_TransmitNextFragment( netchan_t *chan )
 {
 	msg_t send;
-	byte  send_buf[ MAX_PACKETLEN ];
+	byte  send_buf[ MAX_PACKETLEN ] = {0};
```

Comment:
Why is this needed? The code is designed to memset the buffer - see Huff_putBit.

Reply:
Valgrind still complains :shrug: